proxy/proxy.go

```diff
@@ -27,7 +27,7 @@ func RedirectHandler(config *util.Config) http.Handler {
 		t__start := time.Now().UnixNano()
 		domain := util.StripPortFromDomain(req.Host)
 		status := http.StatusOK
-		request_type := "internal"
+		request_type := "unknown"
```

Comment:
Nothing references this value, I suppose it doesn't matter much, but why `unknown`?

Reply:
Well.. There are two if-else branches that overwrites `request_type` in lines 55 and 59 respectively. If none of these branches are hit, request_type will be "unknown". I believe initializing it to internal before was a mistake.